Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
      *
      * @param {string|Object} subject The html selector or object on which function should be applied
      */
-    setup_checkbox_range_selection: function(subject) {
-        Admin.log('[core|setup_checkbox_range_selection] configure checkbox range selection on', subject);
+    "setup_checkbox_range_selection" (subject) {
+
+        Admin.log("[core|setup_checkbox_range_selection] configure checkbox range selection on", subject);
 
-        var previousIndex,
+        let previousIndex,
             useICheck = window.SONATA_CONFIG && window.SONATA_CONFIG.USE_ICHECK
         ;
 
         // When a checkbox or an iCheck helper is clicked
-        jQuery('tbody input[type="checkbox"], tbody .iCheck-helper', subject).click(function (event) {
-            var input;
+        jQuery("tbody input[type=\"checkbox\"], tbody .iCheck-helper", subject).click(function (event) {
+
+            let input;
 
             if (useICheck) {
-                input = jQuery(this).prev('input[type="checkbox"]');
+
+                input = jQuery(this).prev("input[type=\"checkbox\"]");
+
             } else {
+
                 input = jQuery(this);
+
             }
 
             if (input.length) {
-                var currentIndex = input.closest('tr').index();
+
+                const currentIndex = input.closest("tr").index();
 
                 if (event.shiftKey && previousIndex >= 0) {
-                    var isChecked = jQuery('tbody input[type="checkbox"]:nth(' + currentIndex + ')', subject).prop('checked');
+
+                    const isChecked = jQuery(`tbody input[type="checkbox"]:nth(${currentIndex})`, subject).prop("checked");
 
                     // Check all checkbox between previous and current one clicked
-                    jQuery('tbody input[type="checkbox"]', subject).each(function (i, e) {
+                    jQuery("tbody input[type=\"checkbox\"]", subject).each((i, e) => {
+
                         if (i > previousIndex && i < currentIndex || i > currentIndex && i < previousIndex) {
+
                             if (useICheck) {
-                                jQuery(e).iCheck(isChecked ? 'check' : 'uncheck');
+
+                                jQuery(e).iCheck(isChecked ? "check" : "uncheck");
```

Comment:
- eslint: Line 182 exceeds the maximum line length of 80.
- eslint: Ternary operator used.
- eslint: Expected newline between test and consequent of ternary expression.
- eslint: Expected newline between consequent and alternate of ternary expression.